### ISSUE TITLE
Taxi Fix - New Error

### ISSUE THREAD

**Issue by behub-au** [Hekili/hekili#6]
build: 7.3.5.18-2-gaf14c9b just pushed through on curse with the taxi changes is throwing a lua error when you get on a taxi and then when you get off the taxi. Need to Disable and Enable to get the icons back.

```
4x Hekili\Utils.lua:238: attempt to call upvalue 'func' (a nil value)
Hekili\Utils.lua:238: in function <Hekili\Utils.lua:237>

Locals:
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'func' (a nil value)"
func = nil
args = <table> {
}
```

**Comment by behub-au** [comment]
Other info from addon:

> build: 7.3.5.18-2-gaf14c9b
> level: 110
> class: SHAMAN
> spec: enhancement
> 
> talents: earthen_spike
>     fury_of_air
>     landslide
>     lightning_surge_totem
>     overcharge
>     tempest
>     wind_rush_totem
> 
> traits: alpha_wolf=1
>     concordance_of_the_legionfall=28
>     crashing_hammer=5
>     doom_vortex=1
>     doom_winds=1
>     doom_wolves=1
>     earthshattering_blows=1
>     elemental_healing=4
>     forged_in_lava=4
>     gathering_of_the_maelstrom=4
>     gathering_storms=6
>     hammer_of_storms=1
>     lashing_flames=1
>     might_of_the_earthen_ring=1
>     raging_storms=1
>     spirit_of_the_maelstrom=5
>     spiritual_healing=5
>     stormflurry=1
>     unleash_doom=1
>     weapons_of_the_elements=5
>     wind_strikes=4
>     wind_surge=4
>     winds_of_change=1
> 
> sets/legendaries/artifacts: amanthuls_vision=1
>     doomhammer=1
>     eonars_compassion=1
>     smoldering_heart=1
>     soul_of_the_farseer=1
>     tier21=4
> 
> gear: 152169/headdress_of_venerated_spirits/Headdress of Venerated Spirits=1
>     151973/collar_of_nullflame/Collar of Null-Flame=1
>     152004/pauldrons_of_the_soulburner/Pauldrons of the Soulburner=1
>     152166/robes_of_venerated_spirits/Robes of Venerated Spirits=1
>     139077/bramblemail_belt/Bramblemail Belt=1
>     152170/leggings_of_venerated_spirits/Leggings of Venerated Spirits=1
>     134210/tideskorn_sabatons/Tideskorn Sabatons=1
>     152008/realitysplitting_wristguards/Reality-Splitting Wristguards=1
>     151819/smoldering_heart/Smoldering Heart=1
>     152284/zealous_tormentors_ring/Zealous Tormentor's Ring=1
>     151647/soul_of_the_farseer/Soul of the Farseer=1
>     151968/shadowsinged_fang/Shadow-Singed Fang=1
>     154172/amanthuls_vision/Aman'Thul's Vision=1
>     152167/drape_of_venerated_spirits/Drape of Venerated Spirits=1
>     128819/doomhammer/Doomhammer=1
>     128873/fury_of_the_stonemother/Fury of the Stonemother=1

**Comment by Hekili** [comment]
Fixed in 3aa6d1b.
